packages/jobs/lib/runner/render.ts

```diff
@@ -34,8 +35,11 @@ export const renderNodeProvider: NodeProvider = {
                 ownerId,
                 image: { ownerId, imagePath: node.image },
                 serviceDetails: {
-                    env: 'image',
-                    plan: getPlan(node)
+                    runtime: 'image',
+                    plan: getPlan(node),
+                    ...(node.image.includes('-runner')
+                        ? {}
+                        : { envSpecificDetails: { dockerCommand: 'node packages/runner/dist/app.js 80 dockerized-runner' } })
                 },
                 envVars: [
                     { key: 'PORT', value: '80' },
```

Comment:
you could have a `port` variable and use it here and in the dockerCommand so they are always in sync

Reply:
good point